public/user_bio.html

```diff
@@ -0,0 +1,149 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <meta name="author" content="">
+
+    <title>Fam Community App</title>
+
+    <!-- Bootstrap core CSS -->
+    <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
+
+    <!-- Custom fonts for this template -->
+    <link href="vendor/font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic" rel="stylesheet" type="text/css">
+
+    <!-- Plugin CSS -->
+    <link href="vendor/magnific-popup/magnific-popup.css" rel="stylesheet" type="text/css">
+
+    <!-- MAIN CSS FILE TO EDIT -->
+    <link href="vendor/freelancer/css/freelancer.css" rel="stylesheet">
+
+  </head>
+
+  <body id="page-top">
+
+    <!-- Navigation -->
+    <nav class="navbar navbar-expand-lg bg-secondary fixed-top text-uppercase" id="mainNav">
+      <div class="container">
+        <a class="navbar-brand js-scroll-trigger" href="https://fam-io.herokuapp.com/">Fam</a>
+
+
+        <button class="navbar-toggler navbar-toggler-right text-uppercase bg-primary text-white rounded" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+          Menu
+          <i class="fa fa-bars"></i>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarResponsive">
+          <ul class="navbar-nav ml-auto">
+            <li class="nav-item mx-0 mx-lg-1">
+              <a class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger" href="https://fam-io.herokuapp.com/">Home</a>
+            </li>
+            <li class="nav-item mx-0 mx-lg-1">
+              <a class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger" href="https://fam-io.herokuapp.com/signup">LOGIN</a>
+            </li>
+            <li class="nav-item mx-0 mx-lg-1">
+              <a class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger" href="#contact">Contact</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+
+    <!-- GREEN AREA -->
+    <section class="bg-primary text-white mb-0 mt-4" id="about">
+      <div class="container">
+        <h2 class="text-center text-uppercase text-white">Bio</h2>
+        <hr class="star-light mb-5"/>
+          <div class="container mb-4"></div>
+        <form action="#" method="POST" target="self">
+          Bio : <input type="text" name="bio"/>
+          City : <input type="text" name="city"/>
+          facebook: <input type ="url" name="facebook-url"/>
+          Instagram: <input type ="url" name="facebook-url"/>
+        </form>
+          <button class="navbar-toggler navbar-toggler-right text-uppercase bg-primary text-white rounded" type="button">
```

Comment:
the submit button doesn't look like a button and should maybe be lower?

Reply:
Submit button should send to '/signup3' and update db based on local storage email (see @Brechy's PR #42)

btw you should've just kept working off of #40 instead of opening a new PR, for the future.